crates/symbolicli/src/main.rs

```diff
@@ -694,12 +626,19 @@ mod event {
         #[serde(default)]
         post_context: Vec<String>,
 
+        module: Option<String>,
+
         source_link: Option<String>,
 
         in_app: Option<bool>,
 
         #[serde(default)]
         trust: FrameTrust,
+
+        token_name: Option<String>,
```

Comment:
That is not a "public" field though, its only internal. I would say to just make them `None` in `to_js_frame`.